src/sentry/integrations/bitbucket_server/client.py

```diff
@@ -168,3 +183,27 @@ def get_auth(self):
             signature_method=SIGNATURE_RSA,
             signature_type="auth_header",
         )
+
+    def _get_values(self, uri, params):
+        values = []
+        done = False
+        start = 0
+
+        logger.info("Loading values for paginated uri %s", uri)
+
+        while not done:
+            logger.debug("Loading values for paginated uri %s starting from %s", uri, start)
+            new_params = params
+            new_params["start"] = start
+            data = self.get(uri, auth=self.get_auth(), params=new_params)
+            logger.debug("%s values loaded", len(data["values"]))
+
+            values = values + data["values"]
+
+            if "isLastPage" not in data or data["isLastPage"]:
+                logger.debug("Reached last page for paginated uri %s", uri)
+                done = True
```

Comment:
@marbon87 Probably cleaner to just return `values` here. Also, instead of doing a while loop, it might be safer to do a large for loop in case for some reason we never get to the last page. I would prefer not to have an infinite loop if we can avoid it.